src/components/Shop/Modal/Purchase.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext, FC } from "react";
```

Comment:
This component should not exist. It is basically the Button component, unless I am mistaken.